libs/sdk-backend-bear/src/backend/workspace/elements/elements.ts

```diff
@@ -100,4 +130,71 @@ class BearWorkspaceElementsQuery implements IElementQuery {
                 : () => Promise.resolve(emptyResult),
         };
     }
+
+    private createFilterExpressionFromAttributeFilters(attributeFilters: IElementQueryAttributeFilter[]) {
+        interface IFiltersGroupedByUri {
+            [overAttributeUri: string]: IElementQueryAttributeFilter[];
+        }
+
+        const groupsByConnectingAttribute = attributeFilters.reduce<IFiltersGroupedByUri>(
+            (groups, filter) => {
+                const { overAttribute } = filter;
+                if (!isUriRef(overAttribute)) {
+                    throw new NotSupported("Identifier refs are not supported in elements attribute filter");
```

Comment:
Why are they not supported? You can convert idRefs to uriRefs if needed with `objRefToUri`.